uuv_dave/launch/plot_manipulator.launch

```diff
@@ -0,0 +1,7 @@
+<!-- rqt_plot for force feedback wrist sensor -->
+<launch>
+    <node name="ft_sensors_plot" pkg="rqt_plot" type="rqt_plot" args="/rexrov/ft_sensor_wrist/wrench/force/x 
+                /rexrov/ft_sensor_wrist/wrench/force/y
```

Comment:
I believe the torques maybe of more practical interest, since that is more typically measured.  I'll take a look at a demo to illustrate.